tasks/main.yml

```diff
@@ -1,5 +1,34 @@
 ---
-# tasks file for debian_repo
+# tasks file for apt_sources
+
+# This only strips the protocol so override as needed.
```

Comment:
Maybe we could move these into a defaults.yml and include that here to keep the flow of main.yml more clear?